### PR TITLE
init repo structure, build steps, docker steps, etc

### PR DESCRIPTION
**Pull request by aristidesstaffieri**
This sets up the project structure, adds Docker and Make files, and related build steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
...
+You will need
+
+- Node (>=18.0): https://nodejs.org/en/download/
+- Yarn (>=v1.22.5): https://classic.yarnpkg.com/en/docs/install
+
+## Development
+`npm i && npm run start`
+
+## Production build
+`npm run build:prod`
```
Comment (piyalbasu) [inline README.md:15]: just to be symmetric, we should probably use the `yarn` cmd's here

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM node:18
+MAINTAINER SDF Wallets Team <wallet-eng@stellar.org>
+
+RUN mkdir -p /app
+WORKDIR /app
```
Comment (piyalbasu) [inline Dockerfile:5]: do we need to install node and yarn and stuff here? example: https://github.com/stellar/laboratory/blob/master/Dockerfile#L11

Comment (aristidesstaffieri) [reply Dockerfile:5]: nope, using the node image so they come built in.

File: README.md (hunk 1 above)
Comment (aristidesstaffieri) [reply README.md]: fixed in ca473683a27810d0e56bbf4c9e6123e5ae5e8782

File: Dockerfile (hunk 2 above)
Comment (piyalbasu) [reply Dockerfile:5]: Ah v cool 👍

**Review by piyalbasu — APPROVED** [review]

**Pull request merged**